spectrum-offchain/src/backlog/persistence.rs

```diff
@@ -68,4 +71,25 @@ where
         })
         .await
     }
+
+    async fn find_orders<F>(&self, f: F) -> Vec<BacklogOrder<TOrd>>
+    where
+        F: Fn(&TOrd) -> bool + Send + 'static,
+    {
+        let db = self.db.clone();
+        spawn_blocking(move || {
+            db.iterator(rocksdb::IteratorMode::Start)
+                .filter_map(|i| {
+                    let (_, v) = i.unwrap();
+                    let b: BacklogOrder<TOrd> = bincode::deserialize(&v).unwrap();
```

Comment:
We can potentially shoot in our leg with this unsafe `.unwrap()` if we decide to store something other than `BacklogOrder<TOrd>` in the future.

Reply:
Removed the `unwrap()`